firebase-vscode/src/workflow.ts

```diff
@@ -299,9 +259,9 @@ export async function setupWorkflow(
     currentFramework = undefined;
     // Note: discover() takes a few seconds. No need to block users that don't
     // have frameworks support enabled.
+    const { useFrameworks } = getSettings();
     if (useFrameworks) {
-      currentFramework =
-        useFrameworks && (await discover(currentOptions.cwd, false));
```

Comment:
Not sure if you know but what was the original use of this?

Reply:
idk - but it's in the if-statement already, so there's really no need for it